pkg/factory/risingwave_object_factory.go

```diff
@@ -1665,6 +1665,15 @@ func (f *RisingWaveObjectFactory) buildPodTemplateFromNodeGroup(component string
 		})
 	}
 
+	// set resource group in compute component.
+	if component == consts.ComponentCompute && nodeGroup.Name != "default" {
```

Comment:
I am not a big fan of the hardcoded `"default"`, but I can't find other places where we used this value.

Reply:
https://github.com/risingwavelabs/risingwave-operator/pull/816#discussion_r1976862793